CHANGELOG.rst

```diff
@@ -1,40 +1,49 @@
 2.9.0.dev
 =========
 
-New Features
-------------
+**New Features**
```

Comment:
Had to use just a bold "title" instead of a proper subtitle because `rst-lint` would complain with:

```
INFO CHANGELOG.rst:1773 Duplicate implicit target name: "new features".
```

:disappointed: